packages/web-components/fast-foundation/src/data-grid/README.md

```diff
@@ -233,7 +233,7 @@ export const myDataGrid = DataGrid.compose({
 | ------------------------ | ------- | ---------------------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------- |
 | `noTabbing`              | public  | `boolean`                    | `false` | When true the component will not add itself to the tab queue. Default is false.                                                                                                                                                                         |                |
 | `generateHeader`         | public  | `GenerateHeaderOptions`      |         | Whether the grid should automatically generate a header row and its type                                                                                                                                                                                |                |
-| `gridTemplateColumns`    | public  | `string`                     |         | String that gets applied to the the css gridTemplateColumns attribute of child rows                                                                                                                                                                     |                |
```

Comment:
This text is generated.  The typo needs to be fixed in the source where it is extracted from - https://github.com/microsoft/fast/blob/3c14603089495a7af8ed5b55122c418b47c45fd5/packages/web-components/fast-foundation/src/data-grid/data-grid.ts#L173